src/mlpack/tests/metric_test.cpp

```diff
@@ -92,50 +90,54 @@ BOOST_AUTO_TEST_CASE(LINFMetricTest)
 
   ChebyshevDistance lMetric;
 
-  BOOST_REQUIRE_CLOSE((double) arma::as_scalar(arma::max(arma::abs(a1 - b1))),
-                      lMetric.Evaluate(a1, b1), 1e-5);
+  REQUIRE((double) arma::as_scalar(arma::max(arma::abs(a1 - b1))) ==
+          Approx(lMetric.Evaluate(a1, b1)).epsilon(1e-7));
 
-  BOOST_REQUIRE_CLOSE((double) arma::as_scalar(arma::max(arma::abs(a2 - b2))),
-                      lMetric.Evaluate(a2, b2), 1e-5);
+  REQUIRE((double) arma::as_scalar(arma::max(arma::abs(a2 - b2))) ==
+          Approx(lMetric.Evaluate(a2, b2)).epsilon(1e-7));
 }
 
 /**
  * Simple test for IoU metric.
  */
-BOOST_AUTO_TEST_CASE(IoUMetricTest)
+TEST_CASE("IoUMetricTest", "[MetricTest]")
 {
   arma::vec bbox1(4), bbox2(4);
   bbox1 << 1 << 2 << 100 << 200;
   bbox2 << 1 << 2 << 100 << 200;
   // IoU of same bounding boxes equals 1.0.
-  BOOST_REQUIRE_CLOSE(1.0, IoU<>::Evaluate(bbox1, bbox2), 1e-4);
+  REQUIRE(1.0 == Approx(IoU<>::Evaluate(bbox1, bbox2)).epsilon(1e-6));
 
   // Use coordinate system to represent bounding boxes.
   // Bounding boxes represent {x0, y0, x1, y1}.
   bbox1 << 39 << 63 << 203 << 112;
   bbox2 << 54 << 66 << 198 << 114;
   // Value calculated using Python interpreter.
-  BOOST_REQUIRE_CLOSE(IoU<true>::Evaluate(bbox1, bbox2), 0.7980093, 1e-4);
+  REQUIRE(IoU<true>::Evaluate(bbox1, bbox2) ==
+          Approx(0.7980093).epsilon(1e-6));
 
   bbox1 << 31 << 69 << 201 << 125;
   bbox2 << 18 << 63 << 235 << 135;
   // Value calculated using Python interpreter.
-  BOOST_REQUIRE_CLOSE(IoU<true>::Evaluate(bbox1, bbox2), 0.612479577, 1e-4);
+  REQUIRE(IoU<true>::Evaluate(bbox1, bbox2) ==
+          Approx(0.612479577).epsilon(1e-6));
 
   // Use hieght - width representation of bounding boxes.
   // Bounding boxes represent {x0, y0, h, w}.
   bbox1 << 49 << 75 << 154 << 50;
   bbox2 << 42 << 78 << 144 << 48;
   // Value calculated using Python interpreter.
-  BOOST_REQUIRE_CLOSE(IoU<>::Evaluate(bbox1, bbox2), 0.7898879, 1e-4);
+  REQUIRE(IoU<>::Evaluate(bbox1, bbox2) ==
+          Approx(0.7898879).epsilon(1e-6));
```

Comment:
I think this (and other lines) can fit in 80 characters, so no need to wrap them. :+1: